extensions/focustask/src/components/search/search-list.tsx

```diff
@@ -0,0 +1,112 @@
+import {
+  Action,
+  ActionPanel,
+  Icon,
+  List,
+  showHUD,
+  showToast,
+  Toast,
+} from "@raycast/api"
+import {createTask} from "api/helpers"
+import {labelForTaskColumn, orderForTaskColumn} from "helpers/focustask"
+import {groupBy, mapValues, sortBy} from "lodash"
+import {useState} from "react"
+import {useFetchLists, useFetchTasks} from "../../api/hooks"
+import {ChecklistListItem} from "./checklist-list-item"
+import {TaskListItem} from "./task-list-item"
+
+export const SearchList = () => {
+  const {isLoading, tasks, error} = useFetchTasks()
+  const {lists} = useFetchLists()
+
+  const [search, setSearch] = useState("")
+
+  const filteredLists = search
+    ? lists.filter((list) =>
+        list.title.toLowerCase().includes(search.toLowerCase()),
+      )
+    : lists
+
+  const filteredTasks = search
+    ? tasks.filter((task) =>
+        task.title.toLowerCase().includes(search.toLowerCase()),
+      )
+    : tasks.filter((task) => task.visibleInDefaultFrame)
+
+  const groupedTasks = mapValues(
+    groupBy(filteredTasks, (task) => task.column),
+    (tasks) => sortBy(tasks, (task) => task.weight),
+  )
+
+  const groupKeys = sortBy(Object.keys(groupedTasks), (key) =>
+    orderForTaskColumn(key),
+  )
+
+  const groups = groupKeys.map((key) => ({key, tasks: groupedTasks[key]}))
+
+  const placeholder = "Search for tasks and lists, or create a new task."
+
+  const handleCreate = async () => {
+    await showToast({
+      style: Toast.Style.Animated,
+      title: "Creating task",
+      message: "This should only take a moment",
+    })
+
+    const response = await createTask({title: search})
+
+    if ("id" in response) {
+      showHUD("Task created")
+    } else {
+      showHUD("Failed to create a task")
+    }
```

Comment:
```suggestion
    if ("id" in response) {
      toast.style = Toast.Style.Success
      toast.title = "Success"
      toast.message = "Task created"
    } else {
      toast.style = Toast.Style.Success
      toast.title = "Failure"
      toast.message = "Failed to create a task"
    }
```

Reply:
After success it would be cool to clear the list and redraw the list with the new item in the list, do you think you can do that?